hypothesis-python/tests/cover/test_stateful.py

```diff
@@ -437,6 +437,12 @@ def fail_fast(self):
     # expanded to 2 variables.
     assert assignment_line == "v1, v2 = state.populate_bundle()"
 
+    # Make sure the printed code can execute
+    with raises(AssertionError):
+        state = ProducesMultiple()
+        v1, v2 = state.populate_bundle()
+        state.fail_fast()
```

Comment:
AIUI, the point of this patch is that the `v1, v2 = state.populate_bundle()` line will execute? So burying it in a `raises()` block seems a bit obscure to me. I’m not sure I’d understand why this code was here without looking at the PR/commit history.

How about:

* Make the comment more explicit: “Make sure we can unpack the MultipleResults object from state.populate_bundle()”
* Move everything but `state.fail_fast()` out of the `with raises(…)` block